### ISSUE TITLE
cf-* files doesn't available in /var/lib/cfengine3/bin

### ISSUE THREAD

**Issue by zloidemon** [Webhuis/Cfengine-debian#29]
I have issue with cf-\* files, they doesn't exists in /var/lib/cfengine3/bin:

```
# cf-agent -B server.local
2013-10-15T05:04:57-0400    error: cf-promises needs to be installed in /var/lib/cfengine3/bin for pre-validation of full configuration
2013-10-15T05:04:58-0400   notice: R: This autonomous node assumes the role of voluntary client
```

**Comment by basvandervlies** [comment]
This is still a discussion point must the binaries be in /usr/bin or /usr/sbin.  /var/lib/cfengine3/bin  now points to /usr/sbin.   For me `/usr/bin` as the cf-tools can be used as 'ordinary ' user.

**Comment by zloidemon** [comment]
This isn't working if you use a `Community version`. I think need more compatibility with `Community version`.

**Comment by basvandervlies** [comment]
@zloidemon we are building the `community version``.  Sorry but can you elaborate what isn't working for you. We want to build a official deb package. We need to follow the debian guide lines for packing.

**Comment by zloidemon** [comment]
Thank you. I see fixed on 31.

**Comment by bahamat** [comment]
Merged #32.
